include/dsn/cpp/service_app.h

```diff
@@ -69,7 +69,7 @@ namespace dsn
 
         const std::string& name() const { return _name; }
 
-        dsn_gpid gpid() const { return _gpid; }
+        dsn_gpid get_gpid() const { return _gpid; }
```

Comment:
let's be consistent with the above. sometimes we use get because the latter is a type name or we want to pair with set, but not in this case.